conf/local.config

```diff
@@ -9,7 +9,7 @@ executor {
 singularity {
     enabled = true
     autoMounts = true
-    cacheDir = '/data/13000026/pipeline/dev/NPM-sample-qc-aux/containers'
+    cacheDir = '/home/ubuntu/NPM-sample-qc/containers'
```

Comment:
is it fine to have the full path hardcoded here ? might be an issue if the code is run in another context ( i.e. no /home/ubuntu )

Reply:
I dunno. I'd give it a pass for now and see if it turns out to be surfaced by someone as a github-issue

Reply:
This file is not used in this branch, therefore I removed it.